tailwind.config.js

```diff
@@ -45,9 +45,10 @@ const grayScale = {
 
 const socialColors = {
   facebook: '#3b5998',
-  twitter: '#00acee',
+  twitter: '#000000',
   instagram: '#dd2a7b',
-  youtube: '#FF0000',
+  youtube: '#ff0000',
+  reddit: '#ff5700',
```

Comment:
Just to double check if these are the correct colors, as in the design inputs from Mike I've noticed #ff4500 for Reddit (instead of #ff5700), and #0f1419 for X (instead of #000000)

Reply:
Good catch, I overlooked that.

For Reddit, I had searched for the color value and apparently found outdated info. Will update that.

@gekim84 From inspecting the values on the brand toolkit page you linked to, the official X brand color is straight black. Did you intentionally choose something off-black?

Reply:
@Scotchester checked with Mike during the stand-up and it turns out that we can use straight black color for X

Reply:
Since I couldn't find an official PDF stating their brand color, I was using the inspect tool in the browser until I saw that the icons were rendered in a different black.

But when I look at other press release about the x icon, they just say that it's black and white.

So let's just go with black 000000

Reply:
Updated the Reddit color.